__tests__/app.test.js

```diff
@@ -27,13 +27,55 @@ describe("/api/topics", () => {
       });
   });
 });
+
 describe("/api", () => {
-  test("responds with status 200 and accurate endpoint documentation", () => {
+  test("200: responds with accurate endpoint documentation", () => {
     return request(app)
     .get("/api")
     .expect(200)
     .then((response)=>{
         expect(response.body).toEqual(endpointsInfo)
     })
   });
-});
\ No newline at end of file
+});
+
+describe("api/articles/:article_id", ()=>{
+  test("200: returns the article with the associated article_id", ()=>{
+    //const articleId = 1
+      return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body })=>{
+          const {article} = body
+          console.log(article);
+          expect(article).toEqual([{
```

Comment:
Consider using a more flexible jest matcher such as .toMatchObject to allow for future expansion